src/interfaces/rest.js

```diff
@@ -55,7 +55,21 @@ module.exports = function(app) {
           path[path.length - 1] === 'meta' &&
           path[0] === 'vessels'
         ) {
-          const meta = getMetadata(path.slice(0, path.length - 1).join('.'))
+          const metaPath = path.slice(0, path.length - 1).join('.')
+          let meta = getMetadata(metaPath)
+
+          if (path[1] === app.selfId) {
+            const defaultsPath = 'vessels.self.' + path.slice(2).join('.')
+            let fromDefaults = _.get(app.deltaCache.defaults, defaultsPath)
+            if (fromDefaults) {
+              if (meta) {
+                meta = JSON.parse(JSON.stringify(meta))
+                _.merge(meta, fromDefaults)
```

Comment:
I'd prefer native js over lodash, to me `{...meta, ...fromDefault}` would be at least as clear and skip the JSON thingie with mutation.